### PR TITLE
Add new landing page

### PR DESCRIPTION
**Pull request by j-blackmore**
- Move image to the background
- Add AND logo to the landing page
- Make landing page resize and mobile friendly
- Refactor Login page
- Fix Answer page
- Use `<Link>` in navbar

### CODE REVIEW

File: src/components/MainPage/AndBanner.js [hunk 1]
```diff
@@ -1,28 +1,17 @@
...
-    AppBar,
-    Button,
-    Typography,
-    Toolbar,
-    IconButton,
-    InputBase
-} from '@material-ui/core';
-import MenuIcon from '@material-ui/icons/Menu';
-import SearchIcon from '@material-ui/icons/Search';
-import logo from '../../images/AND_logo.png';
```
Comment (JacobCrozier) [inline src/components/MainPage/AndBanner.js]: out here spring cleaning

File: src/components/Question/PostButton.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default function PostButton(props) {
             className={`${classes.root} ${props.className}`}
             variant='contained'
         >
-            Post
+            {props.children || 'Post'}
```
Comment (JacobCrozier) [inline src/components/Question/PostButton.js]: this is super cheeky

File: src/components/landing-page/LandingPage.js [hunk 3]
```diff
@@ -1,45 +1,26 @@
...
+export default function LandingPage(props) {
+    const classes = useStyles();
+
+    return (
+        <Box className={clsx(classes.background, 'background')}>
+            <Box className='landing-page-container'>
+                <img className='logo' src={Logo} />
+            </Box>
+        </Box>
+    );
```
Comment (JacobCrozier) [inline src/components/landing-page/LandingPage.js]: bravo

File: src/components/MainPage/AndBanner.js (hunk 1 above)
Comment (j-blackmore) [reply src/components/MainPage/AndBanner.js]: you know me

File: src/components/Question/PostButton.js (hunk 2 above)
Comment (j-blackmore) [reply src/components/Question/PostButton.js]: ;)

**Pull request merged**